service/implementation/auto_request_api/logic_request_by_react.py

```diff
@@ -172,6 +172,7 @@ def afl_teams_statistics(api_data: Dict[str, str]) -> Dict[str, str]:
 
 
 def afl_players(api_data: Dict[str, str]) -> Dict[str, str]:
+    print(api_data)
```

Comment:
that also belongs to "debug" code, which should not be present on the github.

I'll tell you about logging on Friday